api/proto/teleport/workloadidentity/v1/resource_service.proto

```diff
@@ -0,0 +1,82 @@
+// Copyright 2024 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.workloadidentity.v1;
+
+import "google/protobuf/empty.proto";
+import "teleport/workloadidentity/v1/resource.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/workloadidentity/v1;workloadidentityv1pb";
+
+// WorkloadIdentityResourceService provides methods to manage the
+// WorkloadIdentity resource.
+service WorkloadIdentityResourceService {
+  // CreateWorkloadIdentity creates a new workload identity, it will refuse to
+  // create a workload identity if one already exists with the same name.
+  rpc CreateWorkloadIdentity(CreateWorkloadIdentityRequest) returns (teleport.workloadidentity.v1.WorkloadIdentity);
```

Comment:
```suggestion
  rpc CreateWorkloadIdentity(CreateWorkloadIdentityRequest) returns (WorkloadIdentity);
```

Should we hide the package given it's the same package?

Reply:
9bc819a4ce